spec/syntax.md

```diff
@@ -91,150 +91,198 @@ The syntax specification takes into account the following design restrictions:
 
 _This section is non-normative._
 
-### Messages
+### Messages and Patterns
 
-All messages, including simple ones, are enclosed in `{…}` delimiters:
+A **_message_** is the complete template for a specific message formatting request.
 
-    {Hello, world!}
+All _messages_ MUST contain either a _pattern_ or _selectors_. 
```

Comment:
This statement is a bit odd, as _selectors_ includes at least one _pattern_. How about this instead?

```suggestion
All _messages_ MUST contain one or more _patterns_.
```